src/controllers/taskController.js

```diff
@@ -673,6 +675,8 @@ const taskController = function (Task) {
       return;
     }
 
+    /* Remove Cache If Data Exist */
+    cache.removeCache(req.params.id);
```

Comment:
Same thing just to make absolutely sure it is absolutely unique may want to do ("task-" + req.params.id)